docs/user_guide/assessment.rst

```diff
@@ -277,6 +277,9 @@ Base metric                                     :code:`group_min` :code:`group_m
 :func:`.selection_rate`                         .                 .                 Y                  Y
 :func:`.true_negative_rate`                     .                 .                 Y                  Y
 :func:`.true_positive_rate`                     .                 .                 Y                  Y
+:func:`.exposure`                               .                 .                 Y                  Y
+:func:`.utility`                                .                 .                 Y                  Y
```

Comment:
`utility` seems a bit too generic. (I have a feeling @MiroDudik feels the same way...)
Perhaps `ranking_utility`?
To some extent I'm wondering if these should be grouped with the other metrics at all, or whether this deserves its own section with ranking metrics.

Reply:
Since exposure and utility only work for exposure. I also think it is better to call them `ranking_exposure` and `ranking_utility`